game-core/src/main/java/games/strategy/net/nio/ServerQuarantineConversation.java

```diff
@@ -101,7 +121,7 @@ public Action message(final Serializable serializable) {
                     remoteName, channel.socket().getInetAddress(), serverMessenger.getNodes());
           }
           // send the node its assigned name and our name
-          send(new String[] {remoteName, serverMessenger.getLocalNode().getName()});
+          send(new String[] {remoteName, serverMessenger.getLocalNode().getName(), error});
```

Comment:
No idea what's happening here either

Reply:
Server side counterpart of the part that was confusing, see: https://github.com/triplea-game/triplea/pull/4991#discussion_r314853517